test/block-kit/layout-blocks.tsx

```diff
@@ -239,6 +240,65 @@ describe('Layout blocks', () => {
       ).toStrictEqual([section]))
   })
 
+  describe('<Section> with <MrkDwn>', () => {
+    it('outputs a <Section> block wit a <MrkDwn> component and preserves the verbatim prop', () => {
+      expect(
+        JSXSlack(
+          <Blocks>
+            <Section>
+              <MrkDwn verbatim={false}>Hello!</MrkDwn>
+            </Section>
+          </Blocks>
+        )
+      ).toEqual([
+        {
+          type: 'section',
+          fields: [{ type: 'mrkdwn', text: 'Hello!', verbatim: false }],
```

Comment:
Incorrect. `<Mrkdwn>` is composition object for text so the output of `<Mrkdwn>` as a child of `<Section>` would be expected to assign as `text` field instead of a value of `fields`.

```json
[
  {
    "type": "section",
    "text": {
      "text": "Hello!",
      "type": "mrkdwn",
      "verbatim": false
    }
  }
]
```

If we supported fields, it would be like this:

```jsx
<Section>
  <Field>
    <Mrkdwn verbatim={false}>
      example.com
    </Mrkdwn>
  </Field>
</Section>
```

```json
[
  {
    "type": "section",
    "fields": [
      {
        "text": "example.com",
        "type": "mrkdwn",
        "verbatim": false
      }
    ]
  }
]
```